wal/src/message_queue_impl/namespace.rs

```diff
@@ -0,0 +1,714 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+//! Namespace of wal on message queue
+
+use std::{collections::HashMap, fmt, sync::Arc, time::Duration};
+
+use common_types::{table::Location, SequenceNumber};
+use common_util::{define_result, runtime::Runtime};
+use log::{debug, error, info};
+use message_queue::{ConsumeIterator, MessageQueue};
+use snafu::{ensure, Backtrace, OptionExt, ResultExt, Snafu};
+use tokio::sync::RwLock;
+
+use crate::{
+    kv_encoder::LogEncoding,
+    log_batch::{LogEntry, LogWriteBatch},
+    manager::{ReadContext, ReadRequest, RegionId, ScanContext, ScanRequest, WriteContext},
+    message_queue_impl::{
+        config::Config,
+        encoding::MetaEncoding,
+        region::{self, MessageQueueLogIterator, Region},
+    },
+    table_kv_impl::timed_task::{TaskHandle, TimedTask},
+};
+
+#[derive(Debug, Snafu)]
+pub enum Error {
+    #[snafu(display(
+        "Open region failed, namespace:{}, location:{:?}, err:{}",
```

Comment:
Other module's error message is like `Failed to ...`, so it's best this module follow this style.